scripts/stale.py

```diff
@@ -37,19 +37,20 @@ def main():
     open_issues = repo.get_issues(state="open")
 
     for issue in open_issues:
+        print(issue)
         comments = sorted([comment for comment in issue.get_comments()], key=lambda i: i.created_at, reverse=True)
         last_comment = comments[0] if len(comments) > 0 else None
         if (
             last_comment is not None and last_comment.user.login == "github-actions[bot]"
-            and (dt.utcnow() - issue.updated_at).days > 7
-            and (dt.utcnow() - issue.created_at).days >= 30
+            and (dt.utcnow() - issue.updated_at.replace(tzinfo=None)).days > 7
```

Comment:
so the cause is some API changes from GitHub?

Reply:
Yes, I believe so